.github/workflows/combine-prs.yml

```diff
@@ -0,0 +1,38 @@
+# Combine Dependabot PRs on a schedule
+# See https://github.com/github/combine-prs for more information
+
+name: Combine PRs
+
+on:
+  # Dependabot is configured to run every Wednesday,
+  # so we'll combine the PRs on Thursday, at 01:00.
+  # See /.github/dependabot.yml for more information.
+  schedule:
+    - cron: '0 1 * * 4'
+
+  workflow_dispatch: # allows you to manually trigger the workflow
+
+permissions:
+  contents: write # to create a new branch and merge other branches together
+  pull-requests: write # to create a new PR with the combined changes
+  checks: read # to check if CI is passing or not before combining PRs
+
+jobs:
+  combine-prs:
+    runs-on: ubuntu-latest
+    steps:
+      # See https://github.com/github/combine-prs/blob/main/docs/github-app-setup.md
+      - name: Use GitHub App Token
+        uses: wow-actions/use-app-token@d7957e08172ca2e8e49b35b8d266ad585885edc7 # pinned to the hash of v2.0.2 to guarantee immutability
+        id: generate_token
+        with:
+          app_id: ${{ secrets.COMBINE_PRS_APP_ID }}
+          private_key: ${{ secrets.COMBINE_PRS_PRIVATE_KEY }}
```

Comment:
I've added these secrets to the repository. They are tied to the [GitHub App Gu Combine PRs](https://github.com/apps/gu-combine-prs).